### PR TITLE
Add docs for reapply_sysctl TuneD functionality

### PR DESCRIPTION
**Pull request by jmencak**
Also adding a forgotten operand `debug` option.

https://issues.redhat.com/browse/PSAP-600

### CODE REVIEW

File: modules/custom-tuning-specification.adoc [hunk 1]
```diff
@@ -68,13 +68,20 @@ The individual items of the list:
...
+      reapply_sysctl: <bool> <9>
 ----
 <1> Optional.
 <2> A dictionary of key/value `MachineConfig` labels. The keys must be unique.
 <3> If omitted, profile match is assumed unless a profile with a higher priority matches first or `machineConfigLabels` is set.
 <4> An optional list.
 <5> Profile ordering priority. Lower numbers mean higher priority (`0` is the highest priority).
 <6> A TuneD profile to apply on a match. For example `tuned_profile_1`.
+<7> Optional operand configuration.
+<8> Turn debugging on/off for the TuneD daemon: `true/false` (default is `false`).
```
Comment (ahardin-rh) [inline modules/custom-tuning-specification.adoc:83]: Turn debugging on or off for the TuneD daemon. Options are `true` for on or `false` for off. The default is `false`.

File: modules/custom-tuning-specification.adoc [hunk 2]
```diff
@@ -68,13 +68,20 @@ The individual items of the list:
...
+<9> Turn `reapply_sysctl` functionality on/off for the TuneD daemon: `true/false`.
```
Comment (ahardin-rh) [inline modules/custom-tuning-specification.adoc:84]: Turn  `reapply_sysctl` functionality on or off for the TuneD daemon. Options are `true` for on and `false` for off.

**Comment by ahardin-rh** [comment]
@jmencak Thank you! Just some minor suggestions. I will take care of backporting the `debug` option and will tag you for review.
 @wabouhamad Please provide QE review when you're available. Thank you!

**Review by ahardin-rh — COMMENTED** [review]
Just a few suggestions. :+1:

**Comment by jmencak** [comment]
Thank you for the suggestions, @ahardin-rh !  Adjusted according to your recommendation.

**Comment by ahardin-rh** [comment]
@jmencak LGTM! Once we get QE approval. we can merge. Thanks!

**Comment by jmencak** [comment]
@liqcui could you please provide QE review?  Thank you!

**Pull request merged**